app/models/concerns/foreman_openscap/host_extensions.rb

```diff
@@ -30,6 +30,16 @@ def combined_policies
       combined.uniq
     end
 
+    def scap_status_changed?(policy)
+      last_reports = reports_for_policy(policy).take 2
+      return false if last_reports.length != 2
+      !last_reports.first.eql? last_reports.last
+    end
+
+    def reports_for_policy(policy)
```

Comment:
I need reports for host that belong to a certain policy: `policy.arf_reports.select { |report| report.asset = self.asset }`

I made a slight improvement by adding `limit`